sunpy/instr/tests/test_rhessi.py

```diff
@@ -137,7 +137,7 @@ def test_parse_observing_summary_dbase_file_mock():
 
     # We need to mock this test differently for 3.6 vs 3.7
     # Hopefully they backport the fix to 3.6 in the future.
-    if sys.version_info[1] <= 6:
+    if platform.python_version() < '3.7.2':
```

Comment:
Technically the fix was made in 3.7.2 so I just did this awful str comparison.

Reply:
I don't think this will work, I think

```suggestion
    if sys.version_info[2] < 2 or sys.version_info[1] < 7:
```

might work?
```

Reply:
What about: 

`if sys.version_info[2] < 2 and sys.version_info[1] < 7:`